### ISSUE TITLE
[corpus-infra] [explain_error] AutoformerModel and InformerModel fail explain pass

### ISSUE THREAD

**Issue by penguinwu** [penguinwu/oss-model-graph-break-corpus#51]
## Summary

2 models fail the explain pass with model-specific errors (not Dynamo bugs). These are likely our input construction issues, not compiler problems.

**PyTorch:** 2.11.0+cu128

## Affected Models

**AutoformerModel** (eval, train):
```
Dynamo failed to run FX node with fake tensors: call_method view(*(FakeTensor(...
```

**InformerModel** (eval, train):
```
Attention mask should be of size (2, 1, 48, 48), but is torch.Size([2, 1, 96, 96])
```

The InformerModel error is a shape mismatch between the input attention mask and what the model expects during explain. This is likely a corpus input construction bug — we may be generating inputs with the wrong sequence length for the explain pass.

The AutoformerModel error is a fake tensor incompatibility during FX node tracing.

Both models have graph breaks in identify but we cannot analyze them.
